src/routes/RemediationDetails.js

```diff
@@ -175,6 +181,41 @@ class RemediationDetails extends Component {
                                 </GridItem>
                             </Grid>
                         </StackItem>
+                        { localStorage.getItem('remediations:fifi:debug') === 'true' &&
+                            <StackItem>
+                                <Card>
+                                    <CardHeader className='ins-m-card__header-bold'>Playbook Summary</CardHeader>
+                                    <CardBody>
+                                        <Flex className='ins-c-playbookSummary' breakpointMods={ [{ modifier: FlexModifiers.column }] }>
+                                            <Flex className='ins-c-playbookSummary__overview'>
+                                                <FlexItem breakpointMods={ [{ modifier: FlexModifiers['spacer-xl'] }] }>
+                                                    <DescriptionList
+                                                        isBold
+                                                        title='Total systems'>
+                                                        { pluralize(totalSystems, 'system') }
+                                                    </DescriptionList>
+                                                </FlexItem>
+                                            </Flex>
+                                            <DescriptionList className='ins-c-playbookSummary__settings' title='Playbook settings'>
+                                                <Flex>
+                                                    <FlexItem breakpointMods={ [{ modifier: FlexModifiers['spacer-xl'] }] }>
+                                                        Autoreboot:&nbsp;
+                                                        <b> { remediation.auto_reboot && remediation.needs_reboot ? 'Enabled' : 'Disabled' } </b>
+                                                    </FlexItem>
+                                                    <FlexItem>{ pluralize(stats.systemsWithReboot, 'system') } require reboot</FlexItem>
```

Comment:
should "require" also be pluralized?

Reply:
`system(s)` should be plural in front of `require` so it does not need to be pluralized.

Reply:
I meant the case when you only have 1 system - it should say require**s**. Sorry for confusion.

Reply:
Ah yes you're right then.